<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>1620</width>
      <height>780</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>24</number>
    </property>
    <property name="columnCount">
     <number>9</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>System Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Position</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Contracts_Open</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Entry Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Exit Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Trailing Target</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Trailing OK</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Trailing Exit</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>47</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
